kvmagent/kvmagent/test/utils/sharedblock_utils.py

```
@@ -225,14 +225,15 @@ def sharedblock_add_disk(vgUuid=None, hostUuid=None, diskUuid=None, allSharedBlo
     }))
 
 @misc.return_jsonobject()
-def sharedblock_migrate_volumes(vgUuid=None, hostUuid=None, migrateVolumeStructs=None, provisioning=None, volumePath=None, addons={}):
+def sharedblock_migrate_volumes(vgUuid=None, hostUuid=None, migrateVolumeStructs=None, provisioning=None, volumePath=None, addons={}, kvmHostAddons={}):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)